### ISSUE TITLE
Not Able to see banner tab in Organization Setting 

### ISSUE THREAD

**Issue by pranshu33-git** [microsoft/banner-settings-ado-extension#24]
Hi, We are using 2.1910.12302 (Latest) version of Banner Settings in AzureDevops for our organizations (https://slb-it-sandbox.visualstudio.com/ & https://slb-it.visualstudio.com/).

![image](https://user-images.githubusercontent.com/57208443/110466573-f1fd7b80-80fb-11eb-9dc3-11bb65605179.png)

We have observed today that the Banner tab is not appearing for even collection admins.

Are there any patches to recover this?

**Comment by bchase-humana** [comment]
We are 2.1910.12302 (Latest) as well and noticed it is missing too for us. Last update was in 2019. Very possible something changed in the platform that is preventing custom org level extensions from showing up. I have seen the same thing with other extensions.

https://github.com/microsoft/AzureDevOps-WSJF-Extension/issues/26

**Comment by bchase-humana** [comment]
just spoke with MSFT support. There is a bug that will be fixed soon on Azure DevOps that will allow the third party extension menu settings for org level to reveal again.

**Comment by pranshu33-git** [comment]
yes, I've also got to know the fix is merged and will be ready to be deployed soon.

**Comment by SebastianSchuetze** [comment]
seems to be fixed! I think the issue can be closed @pranshu33-git

**Comment by jdshkolnik** [comment]
I'm unable to see it still. I don't understand why this is happening but I've seen it with other extensions, too. Might be related to [this](https://developercommunity.visualstudio.com/t/Organisation-settings-extensions-section/1356582).

**Comment by gitnubster** [comment]
@jdshkolnik your link tells me a fix is released by now.
Also it seems to be fixed for this particular extension by now.

**Comment by davux** [comment]
Is this fixed? I can't see it in org settings.

**Comment by Alexander-Hjelm** [comment]
Closing as the issue appears to have been resolved.
